appconfig/dqengineconfig.csv

```diff
@@ -2,3 +2,8 @@ query,params,proc,querytype,starttime
 tablecount,.z.d-1,`hdb1,table,09:00:00.000000000
 symfilecheck,`sym,`hdb1,other,09:00:00.000000000
 symcount,(`quote;`sym),`hdb1,table,09:00:00.000000000
+infinitycount,(`quote;`sym),`hdb1,table,09:00:00.000000000
+infinitycount,(`quote;`),`hdb1,table,09:00:00.000000000
+nullcount,(`quote;`sym),`hdb1,table,09:00:00.000000000
+nullcount,(`quote;`),`hdb1,table,09:00:00.000000000
```

Comment:
Not that it matters to much because this is in config but as an FYI you can use shorthand time format and give symbols as a list:
```suggestion
nullcount,`quote`,`hdb1,table,09:00
```